daemon/cmdref.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+func genMarkdown(cmd *cobra.Command) {
```

Comment:
We have 3 or 4 files with the same code as this one. Please move it to a different package or create a GH issue to merge all of them.

Reply:
Can you specify which files you are referring to?

Reply:
```
cilium/cmd/cmdref.go
cilium-health/cmd/root.go
daemon/daemon_main.go
```

Reply:
These functions subtly differ in each package; I'd prefer we keep them separate.

Reply:
Created a GH issue for further exploration - https://github.com/cilium/cilium/issues/6192